keyboards/pimoroni/keybow2040/halconf.h

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include_next <halconf.h>
+
+#undef  HAL_USE_I2C
+#define HAL_USE_I2C TRUE
```

Comment:
In `halconf.h` the order of `#define` and `#include_next` should be different (ChibiOS is not consistent here, you just need to remember that for `halconf.h` the defines go first, and for `mcuconf.h` the undef/define pairs go last).
```suggestion
#define HAL_USE_I2C TRUE

#include_next <halconf.h>
```

Reply:
Heh, fixed in round three